vendor/kubespray/.gitlab-ci.yml

```diff
@@ -93,7 +93,7 @@ before_script:
     # Check out latest tag if testing upgrade
     # Uncomment when gitlab kubespray repo has tags
     #- test "${UPGRADE_TEST}" != "false" && git fetch --all && git checkout $(git describe --tags $(git rev-list --tags --max-count=1))
-    - test "${UPGRADE_TEST}" != "false" && git checkout f7d52564aad2ff8e337634951beb4a881c0e8aa6
+    - test "${UPGRADE_TEST}" != "false" && git checkout 8b3ce6e418ccf48171eb5b3888ee1af84f8d71ba
```

Comment:
The upgrade seems to work accross tags. Shouldn't we use tag version instead of master ?

Reply:
We could, but haven't before. Kubespray releases tend to be... somewhat ad-hoc.